src/app/modules/submission/submission.utils.ts

```diff
@@ -1,6 +1,11 @@
 import { differenceBy, intersectionBy, keyBy, uniqBy } from 'lodash'
 import { err, ok, Result } from 'neverthrow'
 
+import {
+  ParsedClearAttachmentResponse,
+  ParsedClearFormFieldResponse,
+} from 'src/types/api'
```

Comment:
/nit: use relative import

Reply:
fixed in [06c5edc](https://github.com/opengovsg/FormSG/pull/6587/commits/06c5edcbdf73f320c4cbd2562172fbc81871bae5)